src/bindgen/utilities.rs

```diff
@@ -225,17 +244,28 @@ impl SynAttributeHelpers for [syn::Attribute] {
     }
 
     fn has_attr_name_value(&self, name: &str, value: &str) -> bool {
-        self.iter().filter_map(|x| x.parse_meta().ok()).any(|attr| {
-            if let syn::Meta::NameValue(syn::MetaNameValue { path, lit, .. }) = attr {
-                if let syn::Lit::Str(lit) = lit {
-                    path.is_ident(name) && (&lit.value() == value)
-                } else {
-                    false
+        self.attr_name_value_lookup(name)
```

Comment:
This doesn't seem to be equivalent to the old code, which has `any()` semantics... But it seems `has_attr_name_value` is unused, so let's remove it?